release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md

```diff
@@ -84,6 +84,10 @@ To view bug fixes, see the [GitHub milestone for 2201.2.0 (Swan Lake)](https://g
 
 #### New features
 
+##### `io` package
+
+- Added the support for datamapping in CSV read/write APIs.
```

Comment:
```suggestion
- Added the support for data mapping in CSV read/write APIs
```

Reply:
updated in [5487afa](https://github.com/ballerina-platform/ballerina-release/pull/2091/commits/5487afa652eda8868da05e9e5fe3723497754972)